### ISSUE TITLE
type of `LintReport` 

### ISSUE THREAD

**Issue by samuelstroschein** [inlang/inlang#1099]
## Context 

Follow up of https://github.com/inlang/inlang/issues/1072

The current type looks as follows:

```ts
export type LintReport = {
	id: LintRule["id"]
	level: LintRule["level"]
	message: string
}
```

### Proposal 

```ts
export type LintReport = {
	id: LintRule["id"]
	level: LintRule["level"]
        message: string
	messageId: Message['id']
	languageTag: LanguageTag
}
```

- fixes https://github.com/inlang/inlang/issues/1072
- enables easy filtering, see #1098 

```ts
lints.filter(report => report.resourceId === "de-DE" && report.messageId === "id.3")
```

**Comment by ivanhofer** [comment]
Maybe rename `LintReport` to something else because `Report` sounds like a summary of all lints and not a single lint.
Maybe `LintRecord`?

**Comment by samuelstroschein** [comment]
> Maybe rename LintReport to something else because Report sounds like a summary of all lints and not a single lint.
Maybe LintRecord? 

Ah I think it's fine. "settings" in plugins bothers me a lot though. Plugin config makes much more sense

**Comment by samuelstroschein** [comment]
will be addressed in #1149
